OWNERS

```diff
@@ -1,13 +1,9 @@
 approvers:
-- NautiluX
-- RiRa12621
-- drewandersonnz
-- c-e-brumm
-- jharrington22
-- rogbas
+- sre-platform-leads
+- cblecker
 - jewzaam
 - mwoodson
```

Comment:
Are cblecker, jewzaam, and mwoodson redundant now that they're included in sre-platform-leads?